index.js

```diff
@@ -14,40 +14,11 @@
  * limitations under the License.
  */
 
-const { helper } = require('./lib/helper');
-const api = require('./lib/api');
-const { Page } = require('./lib/Page');
-for (const className in api) {
-  if (typeof api[className] === 'function')
-    helper.installAsyncStackHooks(api[className]);
-}
+const { initialisePuppeteer } = require('./lib/index');
```

Comment:
We use American English spelling throughout the code base:

```suggestion
const { initializePuppeteer } = require('./lib/index');
```